Add custom entry point to load libraries (via GPI_EXTRA) (#1457)
cocotb/cocotb: documentation/source/newsfragments/1457.feature.rst

> Add ability to provide a custom entry after ``:`` in a form of ``lib_name_X:A_entry_pointX,lib_name_Y:B_entry_point``.

Reviewer: ```suggestion
Add ability to provide a custom entry point after ``:`` in the format ``lib_name_X:A_entry_point,lib_name_Y:B_entry_point``.
```
in/with form of/format? Native speakers needed :)

Reviewer 2: ```suggestion
Add ability to provide custom entry points for libraries in ``GPI_EXTRA``. Entry points can be defined after ``:`` in the format ``lib_name_X:A_entry_point,lib_name_Y:B_entry_point``.
```

The delimiter between each library in the list has changed from ``:`` to ``,``.

> It follows convention used by some simulators to load libraries.

Reviewer 3: ```suggestion
```
This line doesn't need to be in the release notes, lets leave details to the docs as in my suggestion above.

> Add ability to provide custom entry points for libraries in ``GPI_EXTRA``.

Reviewer 4: ```suggestion
Add ability to provide custom entry points for libraries in :envvar:`GPI_EXTRA`.
```

> Add ability to provide custom entry points for libraries in :envvar:`GPI_EXTRA`.

Reviewer 3: Perhaps
```suggestion
The name of the entry point symbol for libraries in :envvar:`GPI_EXTRA` can now be customized.
```

> Entry points can be defined after ``:`` in the format ``lib_name_X:A_entry_point,lib_name_Y:B_entry_point``.

Reviewer 4: You probably don't need this line, the crossref should do it.

Reviewer 3: Agreed, let's keep the newsfragment concise

> The library delimiter between list of libraries has changed from ``:`` to ``,``.

Reviewer 2: ```suggestion
The delimiter between each library in the list has changed from ``:`` to ``,``.
```

> The library delimiter between list of libraries has changed from ``:`` to ``,``.

Author: :+1:
